distribution/docs/src/main/resources/content/_securityFramework/security-rest-intro.adoc

```diff
@@ -73,8 +73,125 @@ To retrieve the subject, the token is sent through Shiro to the STS Realm where
+|                           |          | No   |                |              /--------->|                    +--------------+          |
+|                           |          v      |                |              |          |                                   |          |
+|                           |  +-------+----+ |                |              |          |                                   |          |
+|                    Yes    |  | Is Browser | |                |              |          |                                   |          |
+|                   /-------|--+            | |                |              |          +-----------------------------------+          |
+|                   |       |  +-------+----+ |                |              |                                                         |
+|                   |       |          | No   |                |              |                                                         |
+|                   |       |          v      |                |              |                                                         |
+|                   |       |  +-------+----+ |                |              |                                                         |
+|                   |       |  | Start ECP  | |                |              |                                                         |
+|                   |       |  |c5F5        | |                |              |                                                         |
+|                   |       |  +------------+ |cDEF            |              |                                                         |
+|                   |       +-----------------+----------------+              |                                                         |
+|                   |                   Redirect to IdP                       |                                                         |
+|                   \---------------------------------------------------------/                                                         |
+|                                                                                                                                       |
++---------------------------------------------------------------------------------------------------------------------------------------+
+
+....
+
+The IdP Handler is a configured handler on the Web SSO Filter just like the other handlers in previous diagram.
+The IdP Handler and the Assertion Consumer Service are both part of the IdP client that can be used to interface with any compliant SAML 2.0 Web SSO Identity Provider.
+
+The Metadata Exchange happens asynchronously from any login event.
+The exchange can happen via HTTP, File, or the metadata XML itself can be pasted into the configuration for either the IdP client or the IdP server that the system ships with.
```

Comment:
✏️ The parallelism of this sentence might read better as `...via HTTP or File, or the metadata XML... `